Draft of demos announcement (#94)
front-commerce/developers.front-commerce.com: source/_posts/introducing-cms-demos.md

---
title: "Introducing Front-Commerce CMS demos"
date: 2019-11-20
---

One of the strength of Front-Commerce is that it is platform agnostic: you can virtually connect to any service and display its data directly on your shop's front-end.

But if you are not used to these notions, it might be unclear what does this means. This is why we've added two new demos focused on <abbr title="Content Management System">CMS</abbr> solutions: [WordPress](https://wordpress.demo.front-commerce.com) and [Prismic](https://prismic.demo.front-commerce.com).

<!-- more -->

## What's a CMS and why should I care?

A <abbr title="Content Management System">CMS</abbr> is a tool that focuses on managing your contents. It can bring you features like advanced content edition, publishing and reviews tools, version managements, etc. But at the very least, it is usually better than your e-commerce back office.

This is because an e-commerce solution focuses on what brings you value: managing orders. Some don't even have tools to edit contents like. And the ones that have some are either not that great or expensive (ex: Magento's Page Builder is only available on the Commerce version).

> Using a dedicated <abbr title="Content Management System">CMS</abbr> seems to be the solution. But previously it usually meant that you had to have two websites:

Author: ```suggestion
Using a dedicated <abbr title="Content Management System">CMS</abbr> seems to be an interesting alternative. But previously it usually meant that you had to have two websites:
```
